### ISSUE TITLE
Discovery: Sitewide Crew products maintenance audits

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11470]
## Description
Accessibility is a continuous process, it is never "done" and in order to make sure that our products continue to be compliant with the most recent standards, newest technologies and processes, we should perform maintenance audits on our products.

## Benefits of a maintenance audit
A maintenance audit is an accessibility review of a product or feature done on existing code, rather than done on a product that is just about to launch. These types of audits help us to find any new issues, that might be a result of newer versions of technology, combinations of technology (e.g. different browsers and screen readers) Consistently doing maintenance audits on our products will provide us with an opportunity to surface issues sooner and in a way that we can be proactive about planning and scheduling a fix rather than if a stakeholder or visitor were to bring these to our attention, we tend to need to respond in a more reactive, immediate manner.  

## What to consider

- I need a full list of all Sitewide Crew products
  - This should be able to be pulled from the [VFS Product directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory)
- When was the last time each went through an audit?
  - Platform Collab Cycle?
  - Maintenance Audit?
  - 508 Office audit?

## Acceptance Criteria
- [x] Determine full list of products that should have maintenance audits
- [x] Determine when each product was last audited
- [x] Create process for maintenance audits
- [x] Suggested schedule for future maintenance audits

**Comment by laflannery** [comment]
[Process for maintenance audits](https://docs.google.com/document/d/11St0_u37LUpEBXsxmBN9bGuK5lkuG7sRfPmIF3h4vJc/edit#)
[List of products and suggested schedule](https://docs.google.com/spreadsheets/d/1Y0xldcjBGRwe_cG4d1wyYuu1QRf9D2ZnJF4UM3NGOOo/edit#gid=878630087)
